### PR TITLE
Code checked and fixed

### PR DESCRIPTION
**Pull request by YevheniiVlasenko**
"Train your code quality" assignment. 
Code has been checked and fixed.

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String someString;
+    private int variable;
```
Comment (d1sam) [inline src/main/java/core/basesyntax/HelloWorld.java:9]: Please think about better names for these variables.

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird! "
+                + "Your task is to fix the checkstyle in this file."
+                + " Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
+        System.out.println(hello);
+    }
+
+    private void initializeVariables() {
+        String a = "Hello mates!";
```
Comment (d1sam) [inline src/main/java/core/basesyntax/HelloWorld.java:37]: For string "Hello mates" you should use class level variables, you don't need to create local variable 'a'.

**Review by d1sam — CHANGES_REQUESTED** [review]
Nice try, but you should edit a few things.

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String greetingString;
```
Comment (mnyshenko) [inline src/main/java/core/basesyntax/HelloWorld.java:8]: This variable is unused, please try to find a proper place to use it

**Comment by YevheniiVlasenko** [comment]
So I did something wrong and now the pull request is closed, and it is said that I want to merge my branch to the master branch. I don't know how we came here. 
Should I do everything anew?

**Pull request closed without merging**